src/BinDeps.jl

```diff
@@ -232,6 +177,11 @@ function run(c::Choices)
     end
 end
 
+"""
+    CCompile(src, dest, options, libs) <: BuildStep
+
+Compile C file `src` to `dest`.
```

Comment:
Might be good to provide equivalent pseudocode like `cc src -o dest <options>`, etc.